src/Uno.UI/UI/Xaml/Shapes/Line.iOSmacOS.cs

```diff
@@ -11,6 +11,13 @@ namespace Windows.UI.Xaml.Shapes
 {
 	public partial class Line
 	{
+		public Line()
+		{
+#if __IOS__
+			ClipsToBounds = false;
```

Comment:
If you need that it means there's a bug un the layouter.  Since the layouter is responsible to handler this property, you should not set it here.

Please implement `ICustomClippingElement` instead.

(I know you just moved the code... but it's still problematic)